migrations/01_create_users_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE users(id SERIAL PRIMARY KEY, email VARCHAR(30), password VARCHAR(20), name VARCHAR(30), username VARCHAR(20));
```

Comment:
Appreciate the use of separate migration files to clearly explain some of the SQL used in the project